### ISSUE TITLE
Next batch of fields to add

### ISSUE THREAD

**Issue by drusepth** [indentlabs/notebook#167]
Based on user feedback, I'm compiling a list of link fields to add to content whenever I/someone gets a chance to add them all in. I think we can rely on custom attributes for text fields now, but until you can also add custom link fields, here's what we'll add next:

Universes
- Genre

Locations
- Languages spoken (replace `language` text field, include migration to migrate data over)

Items
- held_by (link to characters for now, add locations after #116)

Groups
- More tab
  - Creatures
  - Languages
  - Magics

Also, it'd be nice if we could create the join tables and related "join" classes automatically from the `relates` method -- but that's an issue for another issue.
